### ISSUE TITLE
Assassin "Steal" Skill

### ISSUE THREAD

**Issue by eliatlarge** [eliatlarge/FEMultiPlayer-V2#5]
Give them the ability to steal weaponry. Just this alone gives Assassins a major niche. We haven't yet thought of a way someone could abuse this... but there's probably a way to abuse this.

**Comment by SapphireTactician** [comment]
Assassins can't / really shouldn't have Steal though, that's the trade-off and tactical choice one has to make for having Lethality. We could however implement the other 2nd tier promotional route for Thief, which is Rogue, who can steal. I feel Steal + Lethality is OP. Or if we're having both, we can nerf Lethality with it's cousin (*used by the Whisper class - 3rd tier thief*)  Bane, which reduces the target to 1 HP.

**Comment by eliatlarge** [comment]
@SapphireTactician its fine. For now, the community has agreed to let them just make this de-equip an enemy unit for a single turn. I'll change the OP to reflect this.

We decided against rogue because in a game with an assassin + swordmaster, having both with similar moves *plus* a utility unit would defeat the purpose of having swordmasters.

**Comment by SapphireTactician** [comment]
>We decided against rogue because in a game with an assassin + swordmaster, having both with similar moves plus a utility unit would defeat the purpose of having swordmasters.

Not necessarily, Swordmasters have the Astra skill and higher Str and Def. If weapon weight ever becomes a thing, they also can wield heavier weapons without a speed penalty than a Rogue can. It's no biggie though, but I'd love to see all class options represented eventually.
